app/views/home/_vertical_menu.html.erb

```diff
@@ -15,8 +15,9 @@
         <% when Menu::Item %>
           <%= menu_secondary_item item %>
         <% when Menu::Divider %>
-          <%= content_tag(:div, "", :class => "divider") %>
-          <%= content_tag(:span, _(item.caption), :class => "nav-item-pf-header") if item.caption %>
+          <%= content_tag(:li, :class => "list-group-item") do %>
+           <%= content_tag(:span, _(item.caption), :class => "vertical-sub-header") if item.caption %></a>
```

Comment:
I see it's not the code that is related to this PR but does _(item.caption) really work? shouldn't it be N_(item.caption)?

Reply:
The string is extracted in `Loader.rb`already, so I guess it's ok to use `_()` here.
There is another translation issue, which I solved [here](https://github.com/theforeman/foreman/pull/5049)